### PR TITLE
Adding pinniped installation docs

### PR DESCRIPTION
**Pull request by dherbrich**
This is for TAP 1.1 only.

### CODE REVIEW

File: authn-authz/pinniped-login.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+1. Generate and distribute kubeconfig to users
+1. Login with provided kubeconfig
+
+## Generate and distribute kubeconfig to users
+
+As an administrator you need to generate the kubeconfig using the following command:
+
+```
+pinniped get kubeconfig --oidc-skip-listen --oidc-skip-browser --kubeconfig-context <your-kubeconfig-context>  > /tmp/concierge-kubeconfig
```
Comment (ohlookadollar) [inline authn-authz/pinniped-login.md:15]: Do we need the --oidc-skip-listen --oidc-skip-browser flags?

File: authn-authz/pinniped-install-guide.md [hunk 2]
```diff
@@ -0,0 +1,222 @@
+# Installing Pinniped on a single cluster
+
+To support authentication on TAP we are installing [Pinniped](pinniped.dev). Inn this guide we install pinniped on a single cluster TAP setup. Pinniped consists of two components that we will deploy into our cluster.
```
Comment (Manifaust) [inline authn-authz/pinniped-install-guide.md:3]:
```suggestion
To support authentication on TAP we are installing [Pinniped](pinniped.dev). In this guide we install pinniped on a single cluster TAP setup. Pinniped consists of two components that we will deploy into our cluster.
```

File: authn-authz/pinniped-login.md (hunk 1 above)
Comment (dherbrich) [reply authn-authz/pinniped-login.md:15]: we dont need it. removing..

**Comment by RichardJJG** [comment]
Is this ready for merging now?

**Comment by dherbrich** [comment]
yes

**Comment by RichardJJG** [comment]
Thanks, @dherbrich!

**Pull request merged**